tests/rptest/perf/openmessaging_perf_test.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import statistics
+
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.services.cluster import cluster
+from rptest.services.openmessaging_benchmark import OpenMessagingBenchmark
+from ducktape.mark import parametrize
+
+
+class RedPandaOpenMessagingBenchmarkPerf(RedpandaTest):
+    # We need to be a little generous with the wait time here as the benchmark
+    # run often takes longer than the configured time. This is often the case
+    # when consumers are catching up with the produced data, when
+    # consumerBacklogSizeGB is configured (check OMB documentation for details).
+    BENCHMARK_WAIT_TIME_MIN = 30
+
+    def __init__(self, ctx):
+        self._ctx = ctx
+        super(OpenBenchmarkTest, self).__init__(test_context=ctx,
+                                                num_brokers=3)
+
+    @cluster(num_nodes=9)
+    @parametrize(driver_idx=3, workload_idx=2)
```

Comment:
Let's use strings instead of indexes: it's easier to search (ctrl-f or `git grep`) and it's easier to understand what they do. Compare 

```
@cluster(num_nodes=9)
@parametrize(driver_idx=3, workload_idx=2)
def test_perf_with_idempotence_and_max_in_flight(...
```

with

```
@cluster(num_nodes=9)
@parametrize(
  driver="ACK_ALL_GROUP_LINGER_1MS_IDEM_MAX_IN_FLIGHT",
  workload="TOPIC1_PART100_1KB_4PROD_1250K_RATE")
def test_perf_with_idempotence_and_max_in_flight(...
```